### ISSUE TITLE
[Feature Request] Search API - site parameter

### ISSUE THREAD

**Issue by nixoeen** [searx/searx#770]
Having a "site" parameter in the Search API makes it easier for other websites to integrate your search into their website. So the following HTML code can be used to integrate a search in a specific website:
```
<form action="https://searx.me/?" method="get">
    <input type="text" name="q" />
    <input type="hidden" name="site" value="mypersonalsite.com" />
    <input type="submit" />
</form>
```

Result would be same as:
https://searx.me/?q=site%3Amypersonalsite.com%20SEARCH_KEYWORD

**Comment by ghost** [comment]
For the browser search bar this works:
https://searx.me/?q=%s%20site%3Amypersonalsite.com
`%s` being the search term.
Maybe there's a way to use this on a website?

**Comment by nixoeen** [comment]
True, and this is how we are using it in http://forum.ubuntu.ir right now, but I had to make another server-side script to create the URL. It is not possible to directly do it with HTML without extra client-side or server-side script.

Adding this feature makes it easier to integrate the search into websites, specially in template codes.

**Comment by dimqua** [comment]
I'm unable to search by multiple websites like this:
https://searx.info/?q=searx+site:wikipedia.org+site:github.com

**Comment by NuLL3rr0r** [comment]
I believe @nixoeen has a valid point. It would be a hassle to implement it without this feature.
